apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/AplCore.java

```diff
@@ -301,30 +294,21 @@ private void startUp() {
                 AddOns.init();
                 Helper2FA.init(databaseManager);
                 aplAppStatus.durableTaskUpdate(initCoreTaskID,  70.1, "Apollo core classes initialization done");
-//signal to API that core is reaqdy to serve requests. Should be removed as soon as all API will be on RestEasy                
+                //signal to API that core is ready to serve requests. Should be removed as soon as all API will be on RestEasy
                 ApiSplitFilter.isCoreReady = true;
 
                 PrunableArchiveMigrator migrator = CDI.current().select(PrunableArchiveMigrator.class).get();
                 migrator.migrate();
                 // start shard process recovery after initialization of all derived tables but before launching threads (blockchain downloading, transaction processing)
                 recoverSharding();
 
+                if(!dexOrderProcessor.isInitialized()){
```

Comment:
I suggest do not throw exception here, but write log with warn level to notify user. DexOrderProcessor is an optional component of apl-core.

Reply:
changed to warning